src/js/actions/datasetActions.js

```diff
@@ -47,6 +51,34 @@ function sortDataset(dsId, field, order) {
   };
 }
 
+/**
+ * Action creator to add filter data transformations to dataset
+ *
+ * @param {number} dsId - Id of the dataset.
+ * @param {string} expression - expression (in JavaScript
+ * syntax) for the filter predicate. The expression language
+ * includes the variable datum, corresponding to the current
+ * data object.
+ * @returns {Object} FILTER_DATASET action with info about
+ * field to be filtered
+ */
+function filterDataset(dsId, expression) {
+  return {
+    type: FILTER_DATASET,
+    id: dsId,
+    expression: expression
+  };
+}
+
+function showExpressionTextbox(dsId, show, time) {
+  return {
+    type: SHOW_EXPRESSION_TEXTBOX,
+    id: dsId,
+    show: show,
+    time: time
+  };
+}
+
```

Comment:
I don't think these two action creators are used anymore? Be sure to clean them up (along with associated variables and exports).